src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+
+## How drop filters work [#drop-rules-work]
+
+A drop filter directs the agent to drop information if it meets a set of conditions. A drop filter contains three parts:
+
+* Events: A stored record from an interaction within your system.
+* Attributes, or columns: A key-value pair attached to data objects. 
+* Regex: A string of characters and operators that corresponds to kinds of information. 
+
+### Events and attributes 
+
+In a typical AI interaction, a prompt or request undergoes certain processes (like embedding) that are recorded as discrete events. For example, let's say a customer requests a street address on file. The prompt is processed by the model, which pulls additional context through various services and databases. Your AI assistant then returns with a response that contains the requested information. 
+
+Below are the different events that occur in a system in an interaction like this: 
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Event
+            </th>
+            <th>
+                Attribute, or column 
+            </th>
+            <th>
+                Description 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                `LlmChatCompletionMessage`
+            </td>
+            <td>
+                `content`
+            </td>
+            <td>
+                Describes when your model generates text to be delivered to your end user. For example, a completion event is triggered when the model generates a text response containing the address. The final message appers as the value for the attribute `content`.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmFeedbackMessage`
+            </td>
+            <td>
+                `messages`
+            </td>
+            <td>
+                INSERT_CELL_CONTENT
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmEmbedding`
+            </td>
+            <td>
+                `input`
+            </td>
+            <td>
+                Indicates when your model uses a vector to interpret words and sentences. It appears when the LLM processes the prompt for the address and again when the model generates the response. The data appears as the value for the attribute `input`. 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmTool`
+            </td>
+            <td>
+                `input` / `output`
+            </td>
+            <td>
+                INSERT_CELL_CONTENT
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+A complete set of drop filters for a piece of sensitive information should include queries for four events: `LlmChatCompletionMessage`, `LlmFeedbackMessage`, `LlmEmbedding`, and `LlmTool`. Your drop filters correspond to event and column pairs in a given row, as seen in the table. 
```

Comment:
This definitely feels like it should be told to customers before the table to give them further context on what they're seeing